ground/gcs/src/plugins/sysalarmsmessaging/sysalarmsmessagingplugin.cpp

```diff
@@ -93,7 +93,7 @@ void SysAlarmsMessagingPlugin::updateAlarms(UAVObject *systemAlarm)
 {
     UAVObjectField *field = systemAlarm->getField(QString("Alarm"));
 
-    for (uint i = 0; i < field->getNumElements(); ++i) {
```

Comment:
The signed/unsigned issue is why I liked auto... I have moved everything to int though so happy to make the iterator vars ints if desired.